### PR TITLE
Roles mg/gordons temp branch

### PR DESCRIPTION
**Pull request by maeved**
US002- Information about roles displayed in band pages and a link is used to redirect user to new page. Information is being pulled from database.

### CODE REVIEW

File: kjourney/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+## Angular material 
+https://www.freakyjolly.com/angular-material-7-8-add-expansion-panel-accordion-in-angular-project-using-material/#more-2551
 # Kjourney
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.17.
+This project waßs generated with [Angular CLI](https://github.com/angular/angular-cli) version 8.3.17.
```
Comment (micalexis) [inline kjourney/README.md]: whats going on here?

File: kjourney/src/app/app.module.ts [hunk 2]
```diff
@@ -31,16 +32,20 @@ import { NavbarjobfamilyComponent } from './navbarjobfamily/navbarjobfamily.comp
     BandsComponent,
     CapabilityComponent,
     JobfamilyComponent,
-    RolesComponent,
     NavbarcapComponent,
-    NavbarjobfamilyComponent
+    NavbarjobfamilyComponent,
```
Comment (micalexis) [inline kjourney/src/app/app.module.ts:41]: NavbarJobFamilyComponent.... follow the same naming conventions

File: kjourney/src/app/role/role.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>role works!</p>
```
Comment (micalexis) [inline kjourney/src/app/role/role.component.html]: leave a TODO to replace

File: kjourney/README.md (hunk 1 above)
Comment (ShaneKainos) [inline kjourney/README.md]: Is this change necessary? Or is this a mistake/automatically changed?

**Review by micalexis — CHANGES_REQUESTED** [review]
what ticket are these changes for? is this a hotfix/story ? name your branch accordingly!

**Review by Conormc98 — CHANGES_REQUESTED** [review]
Remove role component if not in use

File: kjourney/README.md (hunk 1 above)
Comment (maeved) [reply kjourney/README.md]: Letter has been deleted

Comment (maeved) [reply kjourney/README.md]: Letter has been deleted

**Review by ShaneKainos — APPROVED** [review]
Code seems fine, and the functionality is also working. Happy to approve

**Review by Conormc98 — APPROVED** [review]

**Comment by maeved** [comment]
@ChrisThompsonK @micalexis Commits have been squashed

File: ExpressAPI/index.js [hunk 4]
```diff
@@ -177,6 +177,18 @@ app.get('/IsUserValid', function(req,res){
     res.status(200).send({Status:200})
 });
 
+app.get('/getBandRoles/:bandID', function(req, res) {
```
Comment (micalexis) [inline ExpressAPI/index.js]: make your endpoints RESTful

File: kjourney/src/app/bands/bands.component.html [hunk 5]
```diff
@@ -6,24 +6,36 @@ <h1 id="bandTitle" *ngFor="let band of bands">{{ band.name }}</h1>
...
         </tr>
         </thead>
-        <tbody>
-        <td>
-          <tr id="pop-over" *ngFor="let bandRole of bandRoles">
-            <td>{{ bandRole.jobFamilyName }}</td>
-          </tr>
-        </td>
-        <td  style="float: right;">
+        <tbody style="float: right;">
```
Comment (micalexis) [inline kjourney/src/app/bands/bands.component.html]: styling should be handled in the css for the component

**Review by micalexis — CHANGES_REQUESTED** [review]
address comments

**Comment by maeved** [comment]
changes made and resquashed

**Review by micalexis — APPROVED** [review]
